.github/workflows/build.yml

```
@@ -8,6 +8,16 @@ jobs:
     build:
         runs-on: ubuntu-latest
 
+        services:
+            mysql:
+                image: mysql:8.0
+                env:
+                    MYSQL_ROOT_PASSWORD: root
+                    MYSQL_DATABASE: pancake
+                ports:
```

actionlint

could not parse as YAML: yaml: line 17: did not find expected '-' indicator

(syntax-check)